rules/windows/process_creation/proc_creation_win_susp_rundll32_script_run.yml

```diff
@@ -4,10 +4,10 @@ status: experimental
 description: Detects suspicious process related to rundll32 based on arguments
 references:
     - https://gist.github.com/ryhanson/227229866af52e2d963cf941af135a52
-    - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1218.011/T1218.011.md
-author: frack113
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1218.011/T1218.011.md#atomic-test-3---rundll32-execute-vbscript-command-using-ordinal-number
```

Comment:
Hi,
Every time it is possible we use permlink.
We have make a big job to update the permalink.

Otherwise If in a near futur T1218.011 change to T??? the link will be dead.

Thanks

Reply:
T1218.011 is not change but permlink is not update one. (eg. old link have only #12 test and new link have #13 test.)

May this link will be good choice : https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1218.011/T1218.011.md